### ISSUE TITLE
Can't download volume after recovering from snapshot.

### ISSUE THREAD

**Issue by SviridoffA** [apache/cloudstack#9890]
##### ISSUE TYPE
 * Bug Report

##### COMPONENT NAME

~~~
Storage
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.2
~~~

##### CONFIGURATION
Primary storage: Linstor
Hypervisor: KVM

##### OS / ENVIRONMENT
Ubuntu 24.04.1

##### SUMMARY
When we just trying to download volume from newly created vm it's going without problem. But if volume was created from snapshot? we always get "Failed to copy the volume from the source primary storage pool to secondary storage". Looks like it's problem relayted with raw and qcow conversion, because we can download the same volume if we converting it in template? and download it as a template.

##### STEPS TO REPRODUCE

~~~
1. Make vm volume snapshot
2. Create volume from this snapshot
3. Try to download this volume
~~~

##### EXPECTED RESULTS

~~~
Get link to download volume from secondary storage
~~~

##### ACTUAL RESULTS

~~~
"Failed to copy the volume from the source primary storage pool to secondary storage"
~~~

**Comment by rp-** [comment]
I'm pretty sure this was fixed with 3f5a77ef5802871d24777ef8a50f46b952e200a6
At least it works here on my current cluster.

**Comment by rp-** [comment]
The fix will be part of 4.19.2 and 4.20

**Comment by SviridoffA** [comment]
@rp- Great! Thanks for information, so I'll wait for 4.19.2
